test/runtests.jl

```diff
@@ -26,6 +26,18 @@ testfiles = [
   "transforms.jl"
 ]
 
+# For Functor tests in Functional testset
+struct Polynomial{T<:Real}
+  coeffs::Vector{T}
+end
+Polynomial(args::T...) where {T<:Real} = Polynomial(collect(args))
+function (p::Polynomial)(x)
+  n = length(p.coeffs) - 1
+  sum(zip(p.coeffs, 0:n)) do (a, i)
+    a * x^i
+  end
+end
+
```

Comment:
Please move to before the list of test files 👍🏽

Reply:
OK

Reply:
Done